fix documenation error

documentation build is giving an error
https://github.com/hgrecco/pint/actions/runs/14442659170/job/40496081536

```python3
nbsphinx.NotebookError: CellExecutionError in user/numpy.ipynb:
------------------
import xarray as xr

# Load tutorial data
air = xr.tutorial.load_dataset("air_temperature")["air"][0]

# Convert to Quantity
air.data = Q_(air.data, air.attrs.pop("units", ""))

print(air)
print()
print(air.max())
------------------


---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
File ~/work/pint/pint/.pixi/envs/docs/lib/python3.11/site-packages/xarray/tutorial.py:62, in _check_netcdf_engine_installed(name)
     61 try:
---> 62     import scipy  # noqa: F[40](https://github.com/hgrecco/pint/actions/runs/14442659170/job/40496081536#step:4:41)1
     63 except ImportError:

ModuleNotFoundError: No module named 'scipy'

During handling of the above exception, another exception occurred:

ModuleNotFoundError                       Traceback (most recent call last)
File ~/work/pint/pint/.pixi/envs/docs/lib/python3.11/site-packages/xarray/tutorial.py:65, in _check_netcdf_engine_installed(name)
     64 try:
---> 65     import netCDF4
     66 except ImportError as err:

ModuleNotFoundError: No module named 'netCDF4'

The above exception was the direct cause of the following exception:

ImportError                               Traceback (most recent call last)
Cell In[14], line 4
      1 import xarray as xr
      3 # Load tutorial data
----> 4 air = xr.tutorial.load_dataset("air_temperature")["air"][0]
      6 # Convert to Quantity
      7 air.data = Q_(air.data, air.attrs.pop("units", ""))

File ~/work/pint/pint/.pixi/envs/docs/lib/python3.11/site-packages/xarray/tutorial.py:215, in load_dataset(*args, **kwargs)
    178 def load_dataset(*args, **kwargs) -> Dataset:
    179     """
    180     Open, load into memory, and close a dataset from the online repository
    181     (requires internet).
   (...)
    213     load_dataset
    214     """
--> 215     with open_dataset(*args,**kwargs) as ds:
    216         return ds.load()

File ~/work/pint/pint/.pixi/envs/docs/lib/python3.11/site-packages/xarray/tutorial.py:149, in open_dataset(name, cache, cache_dir, engine, **kws)
    1[47](https://github.com/hgrecco/pint/actions/runs/14442659170/job/40496081536#step:4:48)     default_extension = ".nc"
    1[48](https://github.com/hgrecco/pint/actions/runs/14442659170/job/40496081536#step:4:49)     if engine is None:
--> 1[49](https://github.com/hgrecco/pint/actions/runs/14442659170/job/40496081536#step:4:50)         _check_netcdf_engine_installed(name)
    1[50](https://github.com/hgrecco/pint/actions/runs/14442659170/job/40496081536#step:4:51)     path = path.with_suffix(default_extension)
    1[51](https://github.com/hgrecco/pint/actions/runs/14442659170/job/40496081536#step:4:52) elif path.suffix == ".grib":

File ~/work/pint/pint/.pixi/envs/docs/lib/python3.11/site-packages/xarray/tutorial.py:67, in _check_netcdf_engine_installed(name)
     65             import netCDF4
     66         except ImportError as err:
---> 67             raise ImportError(
     68                 f"opening tutorial dataset {name} requires either scipy or "
     69                 "netCDF4 to be installed."
     70             ) from err
     71 if version == 4:
     72     try:

ImportError: opening tutorial dataset air_temperature requires either scipy or netCDF4 to be installed.

You can ignore this error by setting the following in conf.py:

    nbsphinx_allow_errors = True
Error: Process completed with exit code 2.
```

--- a/pint/pint_convert.py
+++ b/pint/pint_convert.py
@@ -13,6 +13,7 @@ from __future__ import annotations
 import argparse
 import contextlib
 import re
+import sys
 from typing import Any
 
 from pint import UnitRegistry
@@ -24,7 +25,7 @@ def _set(ureg: UnitRegistry, key: str, value: Any):
     object.__setattr__(obj, "scale", value)
 
 
-def _define_constants(ureg: UnitRegistry):
+def _define_constants(ureg: UnitRegistry, use_corr: bool):
     # Measured constants subject to correlation
     #  R_i: Rydberg constant
     #  g_e: Electron g factor
@@ -38,7 +39,7 @@ def _define_constants(ureg: UnitRegistry):
     m_e = (ureg._units["m_e"].converter.scale, 0.00000000028e-30)
     m_p = (ureg._units["m_p"].converter.scale, 0.00000000051e-27)
     m_n = (ureg._units["m_n"].converter.scale, 0.00000000095e-27)
-    if args.corr:
+    if use_corr:
         # Correlation matrix between measured constants (to be completed below)
         #          R_i       g_e       m_u       m_e       m_p       m_n
         corr = [
@@ -109,7 +110,7 @@ def _define_constants(ureg: UnitRegistry):
     ureg._build_cache()
 
 
-if __name__ == "__main__":
+def main():
     parser = argparse.ArgumentParser(
         description="Unit converter.", usage=argparse.SUPPRESS
     )
@@ -192,7 +193,7 @@ if __name__ == "__main__":
                 "Failed to import uncertainties library!\n Please install uncertainties package"
             )
 
-        _define_constants(ureg)
+        _define_constants(ureg, args.corr)
 
         num = nq.magnitude
         fmt = fmt
@@ -218,3 +219,7 @@ if __name__ == "__main__":
 
     fmt = "{:" + fmt + "} {:~P}"
     print(("{:} = " + fmt).format(q, nq.magnitude, nq.units))
+
+
+if __name__ == "__main__":
+    sys.exit(main())

--- a/pint/testsuite/benchmarks/test_00_common.py
+++ b/pint/testsuite/benchmarks/test_00_common.py
@@ -4,7 +4,7 @@ import subprocess
 import sys
 
 
-def test_import(benchmark):
+def import_pint():
     # on py37+ the "-X importtime" usage gives us a more precise
     #  measurement of the import time we actually care about,
     #  without the subprocess or interpreter overhead
@@ -16,3 +16,7 @@ def test_import(benchmark):
     field = line.split(b"|")[-2].strip()
     total = int(field)  # microseconds
     return total
+
+
+def test_import(benchmark):
+    benchmark(import_pint)

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -78,6 +78,9 @@ docs = [
 Homepage = "https://github.com/hgrecco/pint"
 Documentation = "https://pint.readthedocs.io/"
 
+[project.scripts]
+pint-convert = "pint.pint_convert:main"
+
 [build-system]
 requires = ["hatchling", "hatch-vcs"]
 build-backend = "hatchling.build"

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -72,6 +72,7 @@ docs = [
   "recommonmark==0.5.0",
   "babel",
   "sciform",
+  "scipy",
 ]
 
 [project.urls]

--- a/requirements_docs.txt
+++ b/requirements_docs.txt
@@ -20,3 +20,4 @@ sphinx_copybutton
 sphinx_design
 typing_extensions
 sciform
+scipy

